web/src/app/administration/components/workbasket-list/workbasket-list.component.ts

```diff
@@ -153,6 +154,12 @@ export class WorkbasketListComponent implements OnInit, OnDestroy {
 
   performFilter(filterBy: WorkbasketQueryFilterParameter) {
     this.filterBy = filterBy;
+    this.domainService
+      .getSelectedDomain()
+      .pipe(take(1))
+      .subscribe((domain) => {
+        this.filterBy.domain = [domain];
+      });
```

Comment:
you could save the domain before overwriting instead of calling the service again.
```
    const domain = this.filterBy.domain;
    this.filterBy = filterBy;
    this.filterBy.domain = domain;
```
Otherwise, you have to call "this.performRequest" inside the subscription to guarantee that domain was set before calling performRequest.